samigo/samigo-app/src/webapp/css/tool_sam.css

```diff
@@ -507,8 +496,8 @@ a.hideDivision {
 }
 
 .answerkeyFeedbackCommentLabel {
-	FONT-WEIGHT: bold;
-    COLOR:#860000;
+    font-weight: bold;
+    color: #860000;
```

Comment:
```suggestion
    color: var(--warnBanner-color);
```

Reply:
I'm not sure about this one. Since it is out-of-scope of this PR, I'm going to skip it for now.